app/static/js/images.js

```diff
@@ -1,6 +1,5 @@
 var photoTimer = 30000;
 var clockTimer = 1000;
```

Comment:
I tend to write time calculations in terms of their parts to ease readability, e.g.:
```javascript
var photoTimer = 30 * 1000;
var clockTimer = 1 * 1000;
```